SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitTestHelper.java

```diff
@@ -0,0 +1,57 @@
+package com.braintreepayments.api;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Provides helper functions to test the SEPA Direct Debit flow in sandbox.
+ */
+public class SEPADirectDebitTestHelper {
+
+    /**
+     * Generates a valid 27-digit IBAN (International Bank Account Number) for testing SEPA Direct Debit flows.
+     * @return a valid IBAN
+     */
+    public static String generateSandboxIBAN() {
```

Comment:
Nitpick - should we add semver note like in this commit on iOS? https://github.com/braintree/braintree_ios/commit/6fe0c36815e5aa7cc1ab531c0f0d57a650151664